packages/boiler-task-selenium/src/selenium-standalone/get-selenium-standalone-options.js

```diff
@@ -0,0 +1,15 @@
+export default function getSeleniumStandaloneOptions() {
+  return {
+    installOpts: {
+      version: '2.48.2'
+    },
+    startOptions: {
+      version: '2.53.0',
+      logLevel: 'verbose',
+      spawnOptions: {
+        logLevel: 'verbose',
+        stdio: 'ignore'
+      }
+    }
+  };
+}
```

Comment:
Sorry, the different version was when I was trying to debug the null session id bug.  I'll fix that.

The reason this is a function is because I wanted to keep each of the test configuration parts (runner options, Selenium options, BrowserStack options) consistently a function so the interface is the same in case we need to mix in some other overrides or something.  But I'll make it an object to simplify it since we don't need to do that here.  :D